src/System.Data.SqlClient/src/System/Data/SqlClient/SqlDataAdapter.cs

```diff
@@ -0,0 +1,172 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Collections;
+using System.ComponentModel;
+using System.Data.Common;
+using System.Diagnostics;
+
+namespace System.Data.SqlClient
+{
+    public sealed class SqlDataAdapter : DbDataAdapter, IDbDataAdapter, ICloneable
```

Comment:
Do you think that SqlAdapter surface can be brought in completely so that the customers can use all the APIs? 
You don't need to address this comment as part of this PR, but I think that you might as well drive this class to completion in case any APIs are left out.